### PR TITLE
Bump pulp dependencies

### PR DESCRIPTION
**Pull request by evgeni**

### CODE REVIEW

File: packages/pulpcore/python-requests/python-requests.spec [hunk 1]
```diff
@@ -2,14 +2,13 @@
...
 Name:           python-%{pypi_name}
-Version:        2.23.0
+Version:        2.24.0
 Release:        1%{?dist}
 Summary:        Python HTTP for Humans
 
 License:        Apache 2.0
 URL:            https://requests.readthedocs.io
 Source0:        https://files.pythonhosted.org/packages/source/r/%{pypi_name}/%{pypi_name}-%{version}.tar.gz
-Patch0:         patch-requests-certs.py-to-use-the-system-CA-bundle.patch
```
Comment (pcreech) [inline packages/pulpcore/python-requests/python-requests.spec:12]: Is this still needed?

File: packages/pulpcore/python-box/python-box.spec [hunk 2]
```diff
@@ -43,10 +43,15 @@ rm -rf %{pypi_name}.egg-info
 %files -n python3-%{srcname}
 %license LICENSE
 %doc README.rst
+%{python3_sitelib}/__pycache__/box.*
+%{python3_sitelib}/box.py
 %{python3_sitelib}/box
```
Comment (ekohl) [inline packages/pulpcore/python-box/python-box.spec:48]: Is this directory still present if there's a `box.py` file? This looks like an odd package

File: packages/pulpcore/python-requests/python-requests.spec (hunk 1 above)
Comment (ekohl) [inline packages/pulpcore/python-requests/python-requests.spec:12]: Is this intended? Can you no longer use the system store or can certifi do this itself now?

Comment (evgeni) [reply packages/pulpcore/python-requests/python-requests.spec:12]: yes, patched certifi is here: https://github.com/theforeman/foreman-packaging/pull/5555

Comment (evgeni) [reply packages/pulpcore/python-requests/python-requests.spec:12]: yes, patched certifi is here: https://github.com/theforeman/foreman-packaging/pull/5555

File: packages/pulpcore/python-box/python-box.spec (hunk 2 above)
Comment (evgeni) [reply packages/pulpcore/python-box/python-box.spec]: fixed (the dir is there, the py is not)

**Comment by evgeni** [comment]
before this gets hidden in the inline comments: this PR requires the patched certifi from https://github.com/theforeman/foreman-packaging/pull/5555

**Comment by evgeni** [comment]
[test rpm]

**Comment by evgeni** [comment]
```
DEBUG util.py:600:    Status code: 404 for http://mirror.centos.org/centos/8/BaseOS/x86_64/os/Packages/libnghttp2-1.33.0-3.el8_2.1.x86_64.rpm (IP: 64.150.179.24)
```

*sigh*

**Comment by pcreech** [comment]
[test rpm]

**Comment by evgeni** [comment]
[test rpm]

**Comment by evgeni** [comment]
[test rpm]

**Comment by evgeni** [comment]
[test rpm]

**Comment by pcreech** [comment]
Repoclosure failures!  Looks like they all built!

**Comment by evgeni** [comment]
Finally. Thanks!

The repoclosure is because certifi (out of #5555) wasn't mashed yet I guess

**Comment by evgeni** [comment]
can I get an ack? :)

**Pull request merged**